extensions/functions_random.yaml

```diff
@@ -0,0 +1,26 @@
+%YAML 1.2
+---
+scalar_functions:
+  -
+    name: random
+    description: >
+      Return a value between 0 (inclusive) and 1 (exclusive).
+
+      Not suitable for cryptographic applications.
+    impls:
+      - deterministic: false
+        return: fp64
+      - args:
+          - value:
+              - name: Seed
```

Comment:
What are the semantics of seed? For example, would `random(5)` return the same random number for every row of the source relation? Or is it designed to be the seed for some process that creates an RNG before executing the "inner loop" of the source data.